tests/example_loader.py

```diff
@@ -0,0 +1,29 @@
+import os
+import yaml
+from datetime import datetime
+import json
+
+current_directory = os.path.dirname(os.path.realpath(__file__))
+
+
+def load_example(path: str):
+    with open(f"{current_directory}/../specification/components/examples/{path}") as f:
+        if path.endswith("yml") or path.endswith("yaml"):
+            return yaml.safe_load(f)
+        else:
+            return f.read().strip()
+
+
+# To convert JSON string to JSON object
+def load_example_json(path: str):
```

Comment:
Can you please merge this to `load_example` function. You have a if condition there to check the file extension and you can cover both yaml and json files in one function. This means the caller (tests) won't care about the format and it should work for both file formats